spec/controllers/agents_controller_spec.rb

```diff
@@ -374,4 +374,14 @@ def valid_attributes(options = {})
       }
     end
   end
+
+  describe "DELETE memory" do
```

Comment:
Might be good to add a defensive test showing that you cannot delete the memory of some other user's agent?